config/locales/en.yml

```diff
@@ -109,6 +109,7 @@ en:
       destroy_alert: It was not possible to delete the taxon
       restore_success: You have sucessfully restored the taxon
       restore_alert: It was not possible to restore the taxon
+      discard_draft_success: You have successfully discarded the draft taxon
```

Comment:
Maybe it's better to call this "deleted" rather than "discarded" to keep everything consistent